draft-ietf-jose-hpke-encrypt.md

```diff
@@ -476,6 +489,20 @@ When using Key Encryption, the strength of the content encryption algorithm shou
 
 This document adds entries to {{JOSE-IANA}}.
 
+## Updates to "JSON Web Key Use" Registry
+
+The "JSON Web Key Use" registry is updated as follows:
+
+   o  Use Member Value: "HPKEauth"
+   o  Use Description: Key for HPKE sender role (authentication)
+   o  Change Controller: IESG
+   o  Specification Document(s): This document
+
+   o  Use Member Value: "HPKErecv"
+   o  Use Description: Key for HPKE receiver role (decapsulation)
```

Comment:
```suggestion
```

The correct key use value for any encryption operation is "enc", which is already defined.